components/select/Select.razor.cs

```diff
@@ -331,6 +328,8 @@ protected async Task OnVisibleChange(bool visible)
             {
                 OnFocus?.Invoke();
 
+                await JsInvokeAsync(JSInteropConstants.AddClsToElement, Ref, "ant-select-focused");
```

Comment:
You can add/delete the class by ClassMapper.

```
bool focused=false;

ClassMapper.If("ant-select-focused",()=>focused);

```

Reply:
Hi ElderJames,

thank you for your feedback, i will take a look into the ClassMapper function and change the code.

Reply:
@ElderJames 
I accidentally made a mistake when merging this branch in my repository with another PR. If you have problems with the merge, you are welcome to close this PR and I will create a new PR from the origin.

There is still much potential for optimization in this module, but I do not want to increase the PR even further.
Currently there are two more bugs in the creation/deletion of tokens. I will create a bug report for each of them.

Reply:
Thank you @ElDiddi !
You can create another PR to fix other problems.
In this PR, you maybe should rebase from the master branch and commit the last commit again.

![image](https://user-images.githubusercontent.com/7550366/97084110-428da400-1647-11eb-898f-1cc4b51c1ceb.png)

Otherwise, this PR will include repeated transfer changes.